debezium-connector-sqlserver/src/main/java/io/debezium/connector/sqlserver/SqlServerConnectorConfig.java

```diff
@@ -243,12 +245,12 @@ public static SnapshotIsolationMode parse(String value, String defaultValue) {
             .withDescription("The timezone of the server used to correctly shift the commit transaction timestamp on the client side"
                     + "Options include: Any valid Java ZoneId");
 
-    public static final Field MAX_LSN_OPTIMIZATION = Field.createInternal("streaming.lsn.optimization")
```

Comment:
Please don't remove it directly but generate a warning in validator saying the option is no longer available.

Reply:
Done.